frontera/contrib/backends/sqlalchemy/components.py

```diff
@@ -79,7 +83,7 @@ def _modify_page(self, obj):
     def _create_page(self, obj):
         db_page = self.model()
         db_page.fingerprint = obj.meta['fingerprint']
-        db_page.url = obj.url
+        db_page.url = obj.url_as_unicode
```

Comment:
We're building a large scale crawling software, and we have to save resources: processor time, storage capacity, network throughput. Before adding a new field, which will consume all these resources we need to have a strong reason why we're doing that. Talking with @redapple in private Slack we didn't found such a strong reason.

In fact Paul thinks that unicode URLs are only needed if they are going to be reviewed by human. In Frontera, this is quite rare.

He suggested to use https://github.com/scrapy/scrapy/blob/master/scrapy/http/request/__init__.py#L53 safe_url_string from w3lib to convert all URLs to ASCII. 

Let's agree, that in Frontera we store and transfer URLs in ASCII encoding. If we receive them in unicode or other encoding we convert them to ASCII.